pkg/controller/secret/secret_controller.go

```diff
@@ -229,6 +229,10 @@ func createPagerdutyRoute(namespaceList []string) *alertmanager.Route {
 		// https://issues.redhat.com/browse/OSD-3326
 		{Receiver: receiverPagerduty, Match: map[string]string{"cluster": "elasticsearch", "prometheus": "openshift-monitoring/k8s"}},
 
+		// Ensure NodeClockNotSynchronising is routed to SRE
+		// https://issues.redhat.com/browse/OSD-8736
+		{Receiver: receiverPagerduty, Match: map[string]string{"alertname": "NodeClockNotSynchronising", "prometheus": "openshift-monitoring/k8s"}},
```

Comment:
This won't change the severity of the alert.. it's still only going to come in as a warning. You may need some changes to create a receiver that actually manipulates the severity label per the card